bootstrap.py

```diff
@@ -268,8 +278,19 @@ def post_json(url, jdata):
         request.add_header("Accept", "application/json")
         request.get_method = lambda: 'POST'
         reply = opener.open(request)
+    except urllib2.HTTPError, e:
+        if e.code == 401:
```

Comment:
422 (unprocessable entity),
500 (ISE),
502,503,504 - bad gateway, service unavail., gateway timeout,...
even this one:  418 - I'm a teapot (not likely, but who knows? :D )
etc.